src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java

```diff
@@ -47,10 +47,9 @@
 	private Patient myPatient;
 	private ArrayList<MovementWard> drugsData;
 	private JDialog dialogDrug;
-	private JTable JtableData;
+	private JTable jTableData;
 	private OhTableDrugsModel<MovementWard> modelMedWard;
 	private OhDefaultCellRenderer cellRenderer = new OhDefaultCellRenderer();
-	private MovWardBrowserManager movManager = Context.getApplicationContext().getBean(MovWardBrowserManager.class);
```

Comment:
I find it is better to have all managers loaded at the begin of the class, wdyt?

Reply:
Sure if it is being used but it appears not in this case so it is just a performance hit, isn't it?

Reply:
it is, at line 71, no?

Reply:
I mean, it was enough to remove line 68

Reply:
Understand now; fixed.

Reply:
alright! thanks!